compiler/main.bal

```diff
@@ -43,9 +43,10 @@ public function main(string[] filenames, *Options opts) returns error? {
         check showTypes([{ filename: filenames[0] }]);
         return;
     }
+    int debugLevel = opts.debugLevel;
     nback:Options nbackOptions = {
         gcName: check nback:validGcName(opts.gc),
-        debugLevel: opts.debug ? nback:DEBUG_BACKTRACE : nback:DEBUG_NONE
+        debugLevel: debugLevel < nback:DEBUG_NONE || debugLevel > nback:DEBUG_FULL ? nback:DEBUG_BACKTRACE : debugLevel
```

Comment:
Write a function nback:validDebugLevel like nback:validGcName.

Reply:
AFAIU, if it's valid, this function should return debugLevel as an int, instead returning a boolean that says it's valid.

This is what I try to capture as the [parse, don't validate](https://docs.google.com/document/d/1Vj_UErPC1DaNnHY92nmq6TlwA5WwmZYkr4xPqP8Lur0/edit#heading=h.oxug7334y2gf) rule in style guide.

Reply:
I would also assume any other input will result in an error.

Reply:
I have created this method to return DEBUG_BACKTRACE for any integer value which does not equal 0 or 2 (Rust also following a similar thing). If the input is 0 or 2, it will return the debugLevel accordingly.